src/web/server/htmlTemplate.ts

```diff
@@ -105,38 +105,45 @@ export const htmlTemplate = ({
 
     // Duplicated prefetch and preconnect tags from DCP:
     // Documented here: https://github.com/guardian/frontend/pull/12935
-
-    // Information on preconnecting:
+    // Preconnect should be used for the most crucial third party domains
+    // "use preconnect when you know for sure that you’re going to be accessing a resource"
+    // - https://www.smashingmagazine.com/2019/04/optimization-performance-resource-hints/
+    // DNS-prefetch should be used for other third party domains that we are likely to connect to but not sure (ads)
+    // Preconnecting to too many URLs can reduce page performance
+    // DNS-prefetch can also be used as a fallback for IE11
+    // More information on preconnecting:
     // https://css-tricks.com/using-relpreconnect-to-establish-network-connections-early-and-increase-performance/
+    // More information on prefetching:
+    // https://developer.mozilla.org/en-US/docs/Web/Performance/dns-prefetch
     const staticPreconnectUrls = [
         `${CDN}`,
         `https://i.guim.co.uk`,
-        `https://interactive.guim.co.uk`,
-        `https://www.google-analytics.com`,
+        `https://securepubads.g.doubleclick.net`,
+        `https://j.ophan.co.uk`,
+        `https://ophan.theguardian.com`,
     ];
 
-    // Information on prefetching:
-    // https://developer.mozilla.org/en-US/docs/Web/Performance/dns-prefetch
     const staticPrefetchUrls = [
-        `${CDN}`,
-        `https://i.guim.co.uk`,
+        ...staticPreconnectUrls,
         `https://api.nextgen.guardianapps.co.uk`,
         `https://hits-secure.theguardian.com`,
-        `https://j.ophan.co.uk`,
-        `https://ophan.theguardian.com`,
+        `https://interactive.guim.co.uk`,
+        `https://ipv6.guim.co.uk`,
         `https://phar.gu-web.net`,
+        `https://static.theguardian.com`,
+        `https://support.theguardian.com`,
+        `https://www.facebook.com`,
```

Comment:
It’s possible without CMP, we won’t be allowed to hit Facebook or google analytics too